content/en/docs/collector/configuration.md

```diff
@@ -546,21 +546,30 @@ service:
 
 ### Telemetry
 
-Telemetry is where the telemetry for the collector itself can be configured. It
+Telemetry is where the telemetry for the Collector itself can be configured. It
 has two subsections: `logs` and `metrics`.
 
 The `logs` subsection allows configuration of the logs generated by the
-collector. By default the collector will write its logs to stderr with a log
-level of `INFO`. You can also add static key-value pairs to all logs using the
+Collector.
+
+By default, the Collector will write its logs to `stderr` with a log level of
+`INFO`. You can also add static key-value pairs to all logs using the
 `initial_fields` section.
-[View the full list of `logs` options here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L41-L97)
+
+View the full list of the `logs` configuration options
+[here](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L41-L97).
 
 The `metrics` subsection allows configuration of the metrics generated by the
-collector. By default the collector will generate basic metrics about itself and
-expose them for scraping at `localhost:8888/metrics`
-[View the full list of `metrics` options here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L99-L111)
+Collector.
+
+By default, the Collector will generate basic metrics about itself and its
+components, and expose them on port `8888` and path `/metrics`, in Prometheus
+format, so that we can scrape them at `http://localhost:8888/metrics`.
```

Comment:
```suggestion
By default, the Collector generates basic metrics about itself and its
components in Prometheus format; you can scrape them at <http://localhost:8888/metrics>.
```

Reply:
Originally, the doc mentions that the port `8888` is exposed locally but this is NOT true after checking out the code [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/otelcol/unmarshaler.go#L63). So I'd suggest to make it clearer that the metrics are exposed at port `8888` for all interfaces by default (which I'm still trying to confirm whether it's aligned with the design in issue [#28861](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/28861)) but can be configured in the `telemetry` section.

So I'd assume that the code is correct and may enhance this further if it's agreeable with something like this:
```
By default, the Collector generates basic metrics about itself and its
components, and exposes them in Prometheus format at `:8888/metrics`
of all network interfaces, which can be further controlled in `telemetry`'s `metrics`
subsection; you can scrape them at `http://localhost:8888/metrics`.
```

Reply:
@jpkrohling @brightzheng100 can this conversation be resolved?